src/components/ChattingBubble/index.tsx

```diff
@@ -0,0 +1,62 @@
+import type { HTMLAttributes, ReactElement } from 'react'
+import styled from '@emotion/styled'
+
+export interface ChattingBubbleProps extends HTMLAttributes<HTMLDivElement> {
+  user: 'sender' | 'receiver'
+  children: string
+}
+
+interface StyledBubbleProps {
+  user: 'sender' | 'receiver'
+}
+
+export const ChattingBubble = ({
+  user,
+  children,
+  ...props
+}: ChattingBubbleProps): ReactElement => {
+  const redex = /\s/gi
+  const BlankChatLength = children?.length
+  const NoBlankchatLength = children?.replace(redex, '').length
```

Comment:
```suggestion
  const blankChatLength = children?.length
  const noBlankchatLength = children?.replace(redex, '').length
```
camel case 컨벤션 맞추는 게 좋을 것 같습니다!